language/parser/parser.go

```
@@ -186,9 +184,7 @@ func parseOperationDefinition(parser *Parser) (ast.Node, error) {
 			Loc:          loc(parser, start),
 		}), nil
 	}
-	if operation, err = parseOperationType(parser); err != nil {
-		return nil, err
-	}
+	operation, err = parseOperationType(parser)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)